v2/formats/standard.json

```diff
@@ -165,6 +165,20 @@
       "id": "standard_26",
       "restriction_id": "standard_ban_list_24_03"
     },
+    {
+      "active": true,
+      "card_pool_id": "rotation_2023",
+      "date_start": "2024-05-31",
+      "id": "standard_27",
+      "restriction_id": "standard_ban_list_24_05"
+    },
+    {
+      "active": false,
```

Comment:
shouldn't 24_09 be active and 24_05 be not active?

Reply:
I meant to respond in here - The new banlist isn't effective until Friday, so this is technically correct until then.